lib/codecs/src/encoding/format/protobuf.rs

```diff
@@ -285,6 +288,27 @@ mod tests {
         .unwrap();
         assert_eq!(Some(11.0), mfield!(message, "d").as_f64());
         assert_eq!(Some(2.0), mfield!(message, "f").as_f32());
+        // integer version
+        let message = encode_message(
+            &test_message_descriptor("Floats"),
+            Value::Object(BTreeMap::from([
+                ("d".into(), Value::Integer(9223372036854775807)),
```

Comment:
Is this [i64](https://doc.rust-lang.org/std/i64/index.html)::[MAX](https://doc.rust-lang.org/std/i64/constant.MAX.html#)?